style.css

```diff
@@ -468,6 +469,30 @@ margin-bottom: 0;
     margin: 1rem;
     border: 10px solid var(--hover-color);
 }
+/* ---------------------image-component-------------- */
+.image-container{
+    display:flex;
+    flex-direction: column;
+    padding:1rem;
+    border: 2px solid var(--off-white);
+    border-radius: 0.5rem;
+}
+.image-feature{
+    padding:1rem;
+    display:flex;
+    align-items: center;
+    align-items: center;
+    justify-content: center;
+}
+
+.img-feat{
+    max-width:80%;
+    height:auto;
+}
```

Comment:
Can please explain what does image-feature and img-feat does? By seeing the classnames it is not clear what styles does they apply

Reply:
.image-feature is used to style the block of image and style it as making its alignment to the center. img-feat used for sizing the image

Reply:
.image-feature is used to style the block of image and style it as making its alignment to the center. img-feat used for sizing the image can I resolve the issue now?

Reply:
ok